static/app/api/urlpatterns.generated.ts

```diff
@@ -0,0 +1,794 @@
+  '/sentry-apps-stats/',
+  '/sentry-apps/',
+  '/sentry-apps/$sentryAppIdOrSlug/',
+  '/sentry-apps/$sentryAppIdOrSlug/api-tokens/',
+  '/sentry-apps/$sentryAppIdOrSlug/api-tokens/$apiTokenId/',
+  '/sentry-apps/$sentryAppIdOrSlug/avatar/',
+  '/sentry-apps/$sentryAppIdOrSlug/components/',
+  '/sentry-apps/$sentryAppIdOrSlug/features/',
+  '/sentry-apps/$sentryAppIdOrSlug/interaction/',
+  '/sentry-apps/$sentryAppIdOrSlug/publish-request/',
+  '/sentry-apps/$sentryAppIdOrSlug/requests/',
+  '/sentry-apps/$sentryAppIdOrSlug/rotate-secret/',
+  '/sentry-apps/$sentryAppIdOrSlug/stats/',
+  '/sentry-apps/$sentryAppIdOrSlug/webhook-requests/',
+  '/shared/groups/$shareId/',
+  '/shared/issues/$shareId/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/alerts-triggered-index/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/alerts-triggered/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/all-unresolved-issues/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/external-teams/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/external-teams/$externalTeamId/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/issue-breakdown/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/issues/old/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/members/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/projects/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/release-count/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/stats/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/time-to-resolution/',
+  '/teams/$organizationIdOrSlug/$teamIdOrSlug/unresolved-issue-age/',
+  '/tempest-ips/',
+  '/uptime-ips/',
+  '/userroles/',
+  '/userroles/$roleName/',
+  '/users/',
+  '/users/$userId/',
+  '/users/$userId/authenticators/',
+  '/users/$userId/authenticators/$authId/',
+  '/users/$userId/authenticators/$authId/$interfaceDeviceId/',
+  '/users/$userId/authenticators/$interfaceId/enroll/',
+  '/users/$userId/avatar/',
+  '/users/$userId/emails/',
+  '/users/$userId/emails/confirm/',
+  '/users/$userId/identities/',
+  '/users/$userId/identities/$identityId/',
+  '/users/$userId/ips/',
+  '/users/$userId/notification-options/',
+  '/users/$userId/notification-options/$notificationOptionId/',
+  '/users/$userId/notification-providers/',
+  '/users/$userId/notifications/',
+  '/users/$userId/notifications/email/',
+  '/users/$userId/organization-integrations/',
+  '/users/$userId/organizations/',
+  '/users/$userId/password/',
+  '/users/$userId/permissions/',
+  '/users/$userId/permissions/$permissionName/',
+  '/users/$userId/permissions/config/',
+  '/users/$userId/regions/',
+  '/users/$userId/roles/',
+  '/users/$userId/roles/$roleName/',
+  '/users/$userId/subscriptions/',
+  '/users/$userId/user-identities/',
+  '/users/$userId/user-identities/$category/$identityId/',
+  '/wizard/',
+  '/wizard/$wizardHash/',
+];
+
+export type MaybeApiPath = KnownApiUrls[number] | (string & {});
```

Comment:
I think we should `export` `KnownApiUrls` from here only and do the rest in `apiDefinition`:

```
import type {KnownApiUrls} from './urlpatterns.generated';

export type ApiPath = KnownApiUrls;
```

we should also be able to remove the `| (string & {})` to get real type-checking instead of type hints. I only did this so that we can work with it even though we don’t have all urls yet. Can be done in a follow-up though.